src/checkbox.scss

```diff
@@ -83,6 +83,21 @@ $block: #{$fd-namespace}-checkbox;
         border-color: var(--sapField_Hover_BorderColor);
       }
     }
+
+    &:focus {
+      outline: none;
+    }
+
+    &:empty {
+      padding: $fd-checkbox-margin;
+      margin: 0;
+
+      &::before,
+      &::after {
+        margin-right: 0;
+        margin-left: 0;
+      }
+    }
```

Comment:
Will `&:empty` work on angular, which adds some comments to template ?

Reply:
Just checked, the `&:empty` psuedo selector does not get applied to the angular checkbox